From 590497d611c407170bdade315230fd0bc89de554 Mon Sep 17 00:00:00 2001
From: Gerrit Pape <pape@smarden.org>
Date: Thu, 3 May 2007 09:25:24 +0000
Subject: [PATCH 07/13] semctl.S, semget.S, semop.s for parisc.

On hppa, the semctl, semget, and semop functions were unavailable. Trying
to use them results in
> undefined reference to `__ipc'
when linking. This can be seen in the ccontrol build log:
<http://buildd.debian.org/fetch.cgi?pkg=ccontrol;ver=0.9.1%2B20060806-2;arch=hppa;stamp=1160949237>
---
 parisc/semctl.S |    3 +++
 parisc/semget.S |    3 +++
 parisc/semop.S  |    3 +++
 3 files changed, 9 insertions(+), 0 deletions(-)
 create mode 100644 parisc/semctl.S
 create mode 100644 parisc/semget.S
 create mode 100644 parisc/semop.S

Index: dietlibc-0.33~cvs20110918/parisc/semctl.S
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ dietlibc-0.33~cvs20110918/parisc/semctl.S	2011-11-03 21:51:56.000000000 +0100
@@ -0,0 +1,3 @@
+#include "syscalls.h"
+
+syscall(semctl,semctl)
Index: dietlibc-0.33~cvs20110918/parisc/semget.S
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ dietlibc-0.33~cvs20110918/parisc/semget.S	2011-11-03 21:51:56.000000000 +0100
@@ -0,0 +1,3 @@
+#include "syscalls.h"
+
+syscall(semget,semget)
Index: dietlibc-0.33~cvs20110918/parisc/semop.S
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ dietlibc-0.33~cvs20110918/parisc/semop.S	2011-11-03 21:51:56.000000000 +0100
@@ -0,0 +1,3 @@
+#include "syscalls.h"
+
+syscall(semop,semop)
